emva1288/process/results.py

```diff
@@ -786,9 +786,83 @@ def s_2_y_dark(self):
         return self.s_2_y_measured_dark - (self.sigma_2_y_stack_dark /
                                            self.spatial['L_dark'])
 
+    @property
+    def s_2_y_cav(self):
```

Comment:
Please add docstrings for the new properties, you have to follow the `..emva1288` directive, so they get properly rendered in the documentation